server/liveblog/themes/themes.py

```diff
@@ -435,29 +435,9 @@ def _save_theme_settings(self, theme, previous_theme):
 
         # Check if theme settings are changed.
         if 'settings' in previous_theme:
-            options = []
-            old_theme_settings = {}
```

Comment:
new defaults should be reflected in previous "defaulted" settings, so we should keep this code,
but change to support new settings also.